web_frontend/osmaxx/excerptexport/models/bounding_geometry.py

```diff
@@ -47,9 +46,9 @@ def __str__(self):
 
 class BBoxBoundingGeometry(BoundingGeometry):
     def __init__(self, *args, **kwargs):
-        attribute_names = ['north', 'east', 'south', 'west']
-        if are_all_keys_in(kwargs, attribute_names) \
-                and not are_all_keys_in(kwargs, ['south_west', 'north_east']):
+        attribute_names = {'north', 'east', 'south', 'west'}
+        if set(attribute_names).issubset(kwargs) \
+                and not {'south_west', 'north_east'}.issubset(kwargs):
```

Comment:
IMO, this make understanding it more difficult, can you elaborate on the reason for changing this?